glif wallet balance panics

```
$ glif wallet balance
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x6 pc=0x100215798]

goroutine 1 [running]:
math/big.(*Float).Float64(0x12b1ba8e0?)
	/usr/local/Cellar/go/1.20.4/libexec/src/math/big/float.go:959 +0x18
github.com/glif-confidential/cli/cmd.logBal({0x100cd704f, 0x5}, 0x0?, {{0xc00003a300?, 0x0?}}, {0x18, 0xc2, 0x1, 0x24, 0x4d, ...})
	/Users/lanzafame/src/github.com/glif-confidential/cli/cmd/agent_wallet_balance.go:84 +0x2d
github.com/glif-confidential/cli/cmd.glob..func16(0x101596f00, {0x100cd67bc?, 0x0?, 0x0?})
	/Users/lanzafame/src/github.com/glif-confidential/cli/cmd/agent_wallet_balance.go:116 +0x28f
github.com/spf13/cobra.(*Command).execute(0x101596f00, {0x1015db100, 0x0, 0x0})
	/Users/lanzafame/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x101599460)
	/Users/lanzafame/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/lanzafame/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/glif-confidential/cli/cmd.Execute()
	/Users/lanzafame/src/github.com/glif-confidential/cli/cmd/root.go:46 +0x25
main.main()
	/Users/lanzafame/src/github.com/glif-confidential/cli/main.go:21 +0x17
```

--- a/cmd/agent_wallet_balance.go
+++ b/cmd/agent_wallet_balance.go
@@ -5,6 +5,7 @@ package cmd
 
 import (
 	"context"
+	"fmt"
 	"log"
 	"math/big"
 	"time"
@@ -47,6 +48,12 @@ func getBalances(
 		bal, err := lapi.WalletBalance(ctx, addr)
 		if err != nil {
 			errCh <- err
+			return
+		}
+		if bal.Int == nil {
+			err = fmt.Errorf("failed to get %s balance", key)
+			errCh <- err
+			return
 		}
 		balDecimal := denoms.ToFIL(bal.Int)
 		balCh <- balance{bal: balDecimal, key: key}
@@ -81,6 +88,10 @@ func getBalances(
 }
 
 func logBal(key util.KeyType, bal *big.Float, fevmAddr address.Address, evmAddr common.Address) {
+	if bal == nil {
+		log.Printf("Failed to get %s balance", key)
+		return
+	}
 	bf64, _ := bal.Float64()
 	log.Printf("%s balance: %.02f FIL", key, bf64)
 }

